src/view/com/pager/TabBar.tsx

```diff
@@ -135,7 +138,8 @@ export function TabBar({
               onLayout={e => onItemLayout(e, i)}
               style={styles.item}
               hoverStyle={pal.viewLight}
-              onPress={() => onPressItem(i)}>
+              onPress={() => onPressItem(i)}
+              accessibilityRole="tab">
```

Comment:
Are there any tab panels we can associate with these tabs?

Reply:
Ideally the panel views would be associated with the tabs (as `tabpanel`s with `aria-labelledby`), but I didn't feel confident adding it (and afraid it'd require adapting multiple places where the component is used); so I kept it simple.

FWIW, not associating a tabpanel is not a violation of the WAI-ARIA specification -- it's a *should* -- and should not be an issue so long the semantics of the page still allow assistive technologies to let the user navigate. It's best practice to associate them, but getting rid of clickable plain divs is already a huge step up. 😅

I think (am pretty sure) there are many accessibility nits and issues throughout the app, so properly doing the association could be handled in future improvements

Reply:
Agreed, it's definitely nice to get some semantics in there without having to do a huge refactor 👍 would be nice to eventually have it all hooked up.